Client/Frontend/Home/TopSites/DataManagement/TopSitesDataAdaptor.swift

```diff
@@ -39,6 +39,8 @@ class TopSitesDataAdaptorImplementation: TopSitesDataAdaptor, FeatureFlaggable,
     // Raw data to build top sites with
     private var historySites: [Site] = []
     private var contiles: [Contile] = []
+    private(set) var defaultSearchEngine: OpenSearchEngine?
+    private let topSitesDataUtility = TopSitesDataUtility()
```

Comment:
Why does this hang around initialized to a value like it's a kid loitering in a chocolate shop? Should it be passed in the initializer, so we can easily test it passing a mocked utility? Or should we not have a variable here at all, and simply access it using `TopSitesDataUtility().rem....` where we need to use it? I think having it as a variable that's initialized and hangs around for no real reason is the one scenario we don't really want.

Reply:
Yeah good point - I'd lean towards the latter with accessing it like `TopSitesDataUtility().rem....`. I'll make the fix!